src/components/LeaderList/LeaderItem/LeaderItem.scss

```diff
@@ -0,0 +1,28 @@
+.leaderItem {
+  // .leaderItem_container
+  &_container {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    //background-color: darkorange;
```

Comment:
All comments which aren't related to the future logic should be removed before PR merge.